device-telemetry/scripts/docker/run

```diff
@@ -26,6 +26,11 @@ run_container() {
         -e PCS_AAD_APPSECRET \
         -e PCS_TELEMETRY_STORAGE_TYPE \
         -e PCS_TSI_FQDN \
+        -e PCS_AZUREBLOB_CONNSTRING \
```

Comment:
I noticed a few inconsistencies with the docker-compose.yml and the run script.
In comparison with the list above, this run script seems to be missing:
- PCS_APPLICATION_SECRET
- PCS_AUTH_REQUIRED
- PCS_CORS_WHITELIST
- PCS_DIAGNOSTICS_WEBSERVICE_URL
- PCS_SOLUTION_WEBSITE_URL

Additionally, the docker-compose.yml file has PCS_SOLUTION_WEBSITE_URL but does not have PCS_SOLUTION_NAME.